README.md

```diff
@@ -1,96 +1,52 @@
-# What is Ripple?
+# `rippled` - The XRP Ledger Server
 
-![Ripple](docs/images/ripple.png)
+The XRP Ledger is a decentralized cryptographic ledger powered by a network of peer-to-peer servers. The XRP Ledger uses a novel Byzantine Fault Tolerant consensus algorithm to settle and record transactions in a secure distributed database without a central operator. The server software that powers the XRP Ledger is called `rippled` and is available in this repository under the permissive [ISC open-source license](LICENSE). The XRP Ledger is the home of XRP, a digital asset designed to bridge the many different currencies in use worldwide.
```

Comment:
Recommend:

# The XRP Ledger
The XRP Ledger is a decentralized cryptographic ledger powered by a network of peer-to-peer servers. The XRP Ledger uses a novel Byzantine Fault Tolerant consensus algorithm to settle and record transactions in a secure distributed database without a central operator.

## XRP
XRP is the public, counterparty-free asset native to the XRP Ledger and is traded on the open-market and available for anyone to access. When the XRP Ledger was created in 2012 with a finite supply of 100 billion units of XRP. Its creators gifted 80 billion XRP to a company, now called [Ripple](https://ripple.com/); Ripple uses these funds as to fulfill its mission: to build of the Internet of Value, by ushering in a world in which money moves as fast and as efficiently as information does today.

## `rippled`
The reference implementation of the protocol behind XRP Ledger is called `rippled` and is available in this repository under the permissive [ISC open-source license](LICENSE).

Reply:
`When the XRP Ledger was created in 2012 with a finite supply of 100 billion units of XRP.` <-- Doesn't sound like a complete sentence to me.

You might also add a few sentences on how the "mainline" ledger is defined (the one that starts in 2012, whose earliest public ledger has the hash `...` and that the servers at `r.ripple.com` are serving) since like any blockchain, it isn't hard to start a new one, like the testnet.